src/globus_cli/exception_handling/hooks.py

```diff
@@ -94,14 +112,20 @@ def consent_required_hook(exception: globus_sdk.GlobusAPIError) -> None:
     """
     Expects an exception with a required_scopes field in its raw_json
     """
-    _concrete_consent_required_hook(
+    ret = _concrete_consent_required_hook(
         exception.message, exception.info.consent_required.required_scopes
     )
+    if ret is not None:
+        click.get_current_context().exit(ret)
 
 
 def _concrete_consent_required_hook(
     message: str | None, required_scopes: list[str]
-) -> None:
+) -> int | None:
```

Comment:
This doesn't return an int -- it returns 255.

Reply:
I would argue it returns any int in the range 0 to 255, inclusive, or None.
That's the interface I'm defining here.
But a range-literal isn't something that the type system provides, so `int` is most appropriate.